### PR TITLE
OpenAPI: OT

### PR DESCRIPTION
**Pull request by markxiong0122**
implemented OAS on origin_trials_api

### CODE REVIEW

File: api/origin_trials_api.py [hunk 1]
```diff
@@ -158,7 +161,8 @@ def do_post(self, **kwargs):
       if gate.state not in (Vote.APPROVED, Vote.NA):
         self.abort(400, 'Unapproved gate found for trial stage.')
 
-    body = self.get_json_param_dict()
+    #TODO(markxiong0122): remove to_dict() when PR#4213 is merged
```
Comment (jrobbins) [inline api/origin_trials_api.py:164]: #4213 has has merged already.

**Review by jrobbins — COMMENTED** [review]
This looks OK to me, but I did not check all the fields in the OT OpenAPI models.  Daniel knows that better.

**Review by DanielRyanSmith — APPROVED** [review]
This looks okay to me as well, but we should test these changes locally to be sure everything looks as expected (I can help with that during deployment).

**Comment by markxiong0122** [comment]
> This looks okay to me as well, but we should test these changes locally to be sure everything looks as expected (I can help with that during deployment).

I didn't find all OT pages so this indeed needs further testing. Thank you for helping with this! Should I merge this PR or wait until we test it locally?

**Pull request merged**